system/jlib/jstring.cpp

```diff
@@ -455,14 +455,14 @@ StringBuffer & StringBuffer::limited_valist_appendf(unsigned szLimit, const char
             if (0 != szLimit && size>szLimit) size = szLimit; // if so, will be last attempt
             if (size>MAX_BUF_SIZE)
             {
-                WARNLOG("StringBuffer::valist_appendf(\"%s\"): vsnprintf exceeds limit (%d)", format, size);
+                WARNLOG("StringBuffer::valist_appendf(\"%s\"): vsnprintf exceeds limit (%zu)", format, size);
```

Comment:
@richardkchapman what version of the windows system do we claim to support?(!)  %zu is supported by vs2015, but not 2013.

Reply:
We require c++11 support  - and %zu is part of the cpp11 standard afaik?

Reply:
It is part of C99, I'm not sure if c++11 requires C99 support.  I think not.